plugins/parsers/influx/influx_upstream/parser_test.go

```diff
@@ -659,7 +659,7 @@ func TestStreamParser(t *testing.T) {
 			for {
 				m, err := parser.Next()
 				if err != nil {
-					if err == ErrEOF {
+					if errors.Is(err, ErrEOF) {
```

Comment:
You could make use of `require.ErrorIs` here (see https://pkg.go.dev/github.com/stretchr/testify/require#ErrorIs), but I don't want to hold up this PR for that small improvement.